runtime/polkadot/src/lib.rs

```diff
@@ -1088,9 +1081,17 @@ impl frame_support::traits::OnRuntimeUpgrade for FixPolkadotCouncilVotersDeposit
 			},
 		);
 
-		frame_support::debug::info!(target: "runtime::polkadot", "updated {} + {} voter's deposit.", updated, skipped);
+		log::info!(target: "runtime::polkadot", "updated {} + {} voter's deposit.", updated, skipped);
 		BlockWeights::get().max_block
 	}
+
+	fn pre_upgrade() -> Result<(), &'static str> {
+		todo!()
```

Comment:
it should have a small comment about why it is not implemented and why it is not important for now.